authentik/core/api/authenticated_sessions.py

```diff
@@ -115,3 +123,14 @@ class AuthenticatedSessionViewSet(
     filterset_fields = ["user__username", "session__last_ip", "session__last_user_agent"]
     ordering = ["user__username"]
     owner_field = "user"
+
+    @action(detail=False, methods=["POST"], pagination_class=None, filter_backends=[])
+    def bulk_delete(self, request: Request) -> Response:
```

Comment:
This should have an `@extend_schema` decorator for correct request and response schema